chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -105,6 +105,10 @@ const REPORT_BANDWIDTH_THRESHOLD_COUNT: usize = 10_000;
 /// How long a peer has to be unreachable, until we prune it from the in-memory graph.
 const PRUNE_UNREACHABLE_PEERS_AFTER: time::Duration = time::Duration::hours(1);
 
+/// Send all partial encoded chunk messages three times.
+/// We send these messages multiple times to reduce the chance that they are lost
```

Comment:
we should probably only send the 'critical' message 3 times (the one that goes to the block producer).

As all others look like overkill.. 

But we can try it out in shardnet first.